src/schema/me/__tests__/followed_shows.test.ts

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable promise/always-return */
+import { runAuthenticatedQuery } from "test/utils"
+import gql from "lib/gql"
+import cityData from "../../../schema/city/cityDataSortedByDisplayPreference.json"
+import { LOCAL_DISCOVERY_RADIUS_KM } from "../../../schema/city/constants"
+
+const BASE_GRAVITY_ARGS = {
+  size: 10,
+  offset: 0,
+  total_count: true,
+}
+
+const stubResolver = () =>
+  Promise.resolve({
+    body: [],
+    headers: {},
+  })
+
+const generate_query = (params = `(first: 10)`) =>
+  gql`
+    {
+      me {
+        followsAndSaves {
+          shows${params} {
+            edges {
+              node {
+                id
+              }
+            }
+          }
+        }
+      }
+    }
+  `
+
+describe("returns followed shows for a user", () => {
+  let followedShowsLoader
+
+  beforeEach(() => {
+    followedShowsLoader = jest.fn(stubResolver)
+  })
+
+  const expectArgs = args =>
+    followedShowsLoader.mock.calls.some(val => args === val)
+
+  it("generates a predictable URL with no parameters", done => {
+    const query = generate_query()
+    runAuthenticatedQuery(query, {
+      followedShowsLoader,
+    }).then(_data => {
+      expectArgs(BASE_GRAVITY_ARGS)
+      done()
+    })
+  })
```

Comment:
This test could be rewritten to be a little easier to read with `async/await` and a better Jest matcher:

```ts
it("generates a predictable URL with no parameters", async () => {
  const query = generate_query()
  const data = await runAuthenticatedQuery(query, {followedShowsLoader})
  expect(followedShowsLoader).toHaveBeenCalledWith(BASE_GRAVITY_ARGS)
}
```